### DISCUSSION TITLE
Clients for 3DS / Wii / Wii U

### DISCUSSION THREAD

**Discussion by yllekz** [jellyfin/jellyfin#8865] [General]
Hi folks, here's a real oddball off-the-wall question.

So, maybe you know maybe you don't, but the 3DS, Wii and Wii U can all be jailbroken and loaded with custom firmware, resulting in FOSS app development being possible on the platforms. All three platforms have plenty of apps on GItHub, including media players (WiiMC-SS, ThirdTube, etc). I was curious if anyone knew of any Jellyfin apps that work on these platforms, or if there was interest in developing them?

**Comment by nielsvanvelzen** [comment]
You can use the browser, I don't see Jellyfin coming to these devices unless someone really wants to spend their time on it.

**Reply by thepwrtank18** [reply]
Jellyfin is unusable on the browser.

**Reply by yllekz** [reply]
The 3DS's browser is far too old to display the Jellyfin UI.

**Reply by LhymeFruit** [reply]
The browsers dont work.

**Comment by AndrewRGist** [comment]
Man I'd love this. I use Jellyfin to house my personal VHS and Laserdisc archive, and the wii would be among the best options for CRT use.

The rokus that have composite do not find Jellyfin in the store, nor does it successfully sideload.

**Comment by Brennocksclan** [comment]
Dsi

**Comment by Handrail9** [comment]
I don't think this will ever happen, but if it does it would be so awesome lmao

**Comment by LhymeFruit** [comment]
I wana say that this would be SUPER helpful.

**Comment by WEEZOOKARP** [comment]
I tried accessing my jellyfin instance on my "new" 3ds, very sad that the browser can't load past the loading animation

**Comment by Handrail9** [comment]
Just thought about this, if you set up [Sunshine](https://app.lizardbyte.dev/Sunshine/?lng=en-US) or [Games on Whales](https://games-on-whales.github.io/), then install [Moonlight](https://moonlight-stream.org/) on your (homebrewed) [Wii U](https://github.com/GaryOderNichts/moonlight-wiiu#quick-start) or [3DS](https://github.com/zoeyjodon/moonlight-N3DS) (No Wii client ATM) you could theoretically stream Jellyfin from a computer/server to your 3DS or Wii U. It's a lot of setup for such a silly idea, but it is in fact, easier than coding an app for Jellyfin from scratch for Wii U/3DS

**Reply by LostShadowGD** [reply]
it may be easier, but it may be less fun...

**Comment by fabienmillet** [comment]
It happened! WiiFin is a Jellyfin client for the Nintendo Wii, just released v0.1.0 today. Supports library browsing, video/music playback via server-side transcoding, HTTPS streaming... Ships as .dol for Homebrew Channel and .wad for a proper Wii channel.
https://github.com/fabienmillet/WiiFin/releases/tag/v0.1.0

**Comment by Handrail9** [comment]
2 clients have popped up for 3DS as well! 
https://github.com/8-bitStudio/3d-jelly
https://github.com/arechawla/3DSfin (n3DS only)

**Reply by LostShadowGD** [reply]
i noticed this too. too bad i'd started making my own (not Jellyfin, but my own thing) already. it was nowhere near done, so i might just use these...

**Comment by Ybd0** [comment]
[removed]

This is an ai coded Jellyfin Client for the Wii U 

Use at your own risk. It works great with 1080p at 8 Mbit/s
